examples/target-group-policy.yaml

```diff
@@ -12,3 +12,11 @@ spec:
     healthCheck:
         enabled: true
         intervalSeconds: 10
+        timeoutSeconds: 1
+        healthyThresholdCount: 3
+        unhealthyThresholdCount: 2
+        path: "/"
+        port: 80
```

Comment:
The port should be 8090 instead in the default setting since it refers to the target port

Reply:
Changed, BTW if  grpcBin's TLS cert cipher is a concern, I could later create my own cert and build my own grpcBin image